Lib9c/TableData/Cost/EnhancementCostSheetV2.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using Nekoyume.Model.Item;
+
+namespace Nekoyume.TableData
+{
+    using static TableExtensions;
+    
+    public class EnhancementCostSheetV2 : Sheet<int, EnhancementCostSheetV2.Row>
+    {
+        public class Row: EnhancementCostSheet.Row
```

Comment:
It is just a comment about style.

```suggestion
        public class Row : EnhancementCostSheet.Row
```